demo-apps/chaos-monkey-demo-app/src/test/java/com/example/chaos/monkey/chaosdemo/controller/HelloControllerTest.java

```diff
@@ -35,7 +37,7 @@ public class HelloControllerTest {
     @Autowired
     private MockMvc mockMvc;
 
-    @MockBean
+    @MockitoBean
```

Comment:
Can you please do this change for the remaining `@MockBean` as well if possible? 🙃🙏